scripts/solve_network.py

```diff
@@ -1016,15 +1091,15 @@ def solve_network(n, config, solving={}, opts="", **kwargs):
     else:
         n_ref = None
 
-    n = prepare_network(n, solving["options"])
+    # needed to get `n.model` property

```

Comment:
needed?

Reply:
It has been needed to use linopy methods as creating PyPSA.Network object doesn't enable them. I'll check it I may be missing something

Reply:
After cross-checking, have found that initialisation is not needed. Not sure why it was needed before. Probably I tested extra-functionalites fetched in some exotic way. Or it was issue fixed with the later releases of linopy and/or PyPSA